### ISSUE TITLE
Ari Boyarsky

### ISSUE THREAD

**Issue by bensoltoff** [uchicago-computation-workshop/2019_spring_conf#20]

**Comment by bensoltoff** [comment]
* Seems like something someone would have attempted already. Good selection of a ripe topic
* Seems like the general arguments of the differences between logistic regression and SVM. Why use SVM vs. another machine learning algorithm like decision trees or neural networks?

**Comment by rickecon** [comment]
* Seems like a novel contribution to an existing method. Looks promising. Have you talked to Heckman?
* Really strong theory that combines analytics with computation.
* You should definitely talk to some Econ or Booth economists (econometricians).

**Comment by ariboyarsky** [comment]
@bensoltoff Thank you for your comments! Surprisingly no, while SVM has been used for propensity score estimation it hasn't been proposed as a possible test of the overlap condition in matching estimators - which is often an overlooked but vital assumption. In general, yes - but I think the more interesting case is in the Roy selection model where the test tends to reject (which we would want) while the logistic regression will accept. The reason to use SVM is that we are interested in the interpretability of the results, especially in selecting observations near the decision hyperplane.  We would lose this in the neural network. Additionally, I would expect the decision tree would perform similarly if it could be formulated to utilize a RKHS approach. There is also an Athey and Imbens 2016 paper that considers trees for counterfactual estimation. An issue this presents is the discreteness in the binning over continuous variables although likely not an empirical issue. Another, more forward looking reason, is that  the SVM can easily be formulated with a check-loss function which may allow us to characterize heterogeneity in treatment uptake. I am working on a paper like this with Prof. Pouliot. 

@rickecon Thank you so much also for your comments! I haven't actually spoken with Prof. Heckman about this however I've discussed this with a few econometricians at Econ and Harris, including of course my advisor Prof. Torgovitsky. I think the next steps would be to consider a more sophisticated conditional probability estimation approach perhaps considering some asymptotic results that recently arose for finite dimensional kernels.
